app/containers/WalletInfo/TokenBalances.jsx

```diff
@@ -26,21 +26,25 @@ const TokenBalances = ({ tokenBalances, showModal }: Props) => (
       </tr>
     </thead>
     <tbody>
-      {tokenBalances.map((token: TokenBalanceType, index: number) => {
+      {!!tokenBalances.length && tokenBalances.map((token: TokenBalanceType, index: number) => {
```

Comment:
if `tokenBalances` is passed as a null or undefined the length check will cause it to die.
you can use lodash `compact` (it will always return an array)
```
import { compact } from 'lodash'
{compact(tokenBalances).map((token: TokenBalanceType, index: number) => {
```

Reply:
Thanks for this @evgenyboxer!